examples/animation/implicit/container1/lib/main.dart

```diff
@@ -1,7 +1,3 @@
-// Copyright 2019 the Dart project authors. All rights reserved.
```

Comment:
I'm curious why this has been deleted. We need all code to have copyright headers.

Reply:
It causes a little confusion with the line numbers with our current diff setup. I can add them back since it's not a major problem.